main/inc/lib/main_api.lib.php

```diff
@@ -7513,3 +7583,15 @@ function api_site_use_cookie_warning_cookie_exist()
 {
     return isset($_COOKIE['ChamiloUsesCookies']);
 }
+
+/**
+ * Verify whether a datetime is valid
+ * @param string $dateTime The datetime to validate
+ * @param string $format
+ * @return boolean
+ */
+function apiIsValidDate($dateTime, $format = 'Y-m-d h:m:i') {
```

Comment:
This function already exists "api_is_valid_date" see the internationalization.lib.php file